src/actions/ast.js

```diff
@@ -85,8 +86,11 @@ export function setOutOfScopeLocations() {
     const source = getSourceFromId(getState(), location.sourceId);
 
     let locations = null;
-    if (location.line && source && isPaused(getState())) {
-      locations = await findOutOfScopeLocations(source.id, location);
+    if (location.line && location.column && source && isPaused(getState())) {
```

Comment:
why do we have to do this?
would `if (location && ..) {}` do the job?

Reply:
Neither are actually necessary to pass typing due to the `any` casting below. However, I added it for clarity because `Location` defines `column` as optional but `AstPosition` defines it as required so I wanted to check before blindly casting.